controller/api/public/stat_summary.go

```diff
@@ -337,9 +337,10 @@ func (s *grpcServer) trafficSplitResourceQuery(ctx context.Context, req *pb.Stat
 					Namespace: tsStats.namespace,
 					Type:      req.GetSelector().GetResource().GetType(),
 				},
-				TimeWindow: req.TimeWindow,
-				Stats:      tsBasicStats[currentLeaf],
-				TsStats:    trafficSplitStats,
+				TimeWindow:     req.TimeWindow,
+				MeshedPodCount: 1,
+				Stats:          tsBasicStats[currentLeaf],
+				TsStats:        trafficSplitStats,
```

Comment:
(1/2) This is the significant part of this PR. The other changes are test files updates that have gotten through their initial merges

Reply:
The fact that we have to set MeshedPodCount to a non-zero value in order for Traffic split rows to be rendered is super non-obvious.  At the very least this should explained in a comment here.  But I think an even clearer solution would be in to the stat command if the resource is a traffic split then we should just set the `unmeshed` option to true.  Omitting unmeshed resources is not a thing that makes sense for traffic splits.

Reply:
@adleong Thanks yea I was thinking a little more about this this weekend and I agree with this as I mentioned in my [comment](https://github.com/linkerd/linkerd2/pull/4273#issuecomment-615517376). This change as it is is not very obvious and I think it would be much simpler to just set `unmeshed` to true if this is a trafficsplit resource.

Reply:
... or more explicitly, leaving options alone, replacing
```go
if !options.unmeshed && r.GetMeshedPodCount() == 0 {
  continue
}
```
with
```go
if r.Resource.Type == k8s.TrafficSplit || (!options.unmeshed && r.GetMeshedPodCount() == 0) {
  continue
}
```

Reply:
@adleong @alpeb I've add the change